modules/ROOT/pages/file/file-documentation.adoc

```diff
@@ -80,110 +75,104 @@ A ConnectionProvider which provides instances of FileSystem from instances of Fi
 
 `<file:copy>`
 
-Copies the file at the sourcePath into the targetPath.
-
-If targetPath doesn't exist, and neither does its parent, then an attempt is made to create depending on the value of the createParentFolder argument. If such an argument occurs, then a FILE:ILLEGAL_PATH is thrown.
+Copies the file or directory specified in *Source Path* into the *Target Path*. The source path can be either a file or a directory. If it points to a directory, then it is copied recursively.
```

Comment:
If the argument createParentFolder is not provided and the path doesn't exist, it will throw a FILE:ILLEGAL_PATH error. This should be documented: https://github.com/mulesoft/mule-file-connector/blob/c1f72b136e999384533cecbba35b843b183ef525/src/main/java/org/mule/extension/file/internal/FileOperations.java#L167